test/format_test.js

```diff
@@ -163,6 +163,50 @@ describe('format utility', function() {
     });
   });
 
+  describe('timeChangeInfo', function() {
+    it('should be a function', function() {
+      assert.isFunction(fmt.timeChangeInfo);
+    });
+
+    it('should error if 2 arguments are not passed', function() {
+      var err = 'You have not provided two datetime strings';
+      var x = '2014-01-01T01:00:00.000Z';
+      expect(fmt.timeChangeInfo.bind(fmt)).to.throw(err);
+      expect(fmt.timeChangeInfo.bind(fmt, x)).to.throw(err);
+    });
+
+    it('should return an object containing strings of times when both are on same day', function() {
+      var x = '2014-01-01T01:00:00.000Z';
+      var y = '2014-01-01T04:00:00.000Z';
```

Comment:
I've adjusted but causes failure. Not sure why right now as test results are very unhelpful! Will investigate further.